app/assets/stylesheets/govuk_publishing_components/components/_big-number.scss

```diff
@@ -1,13 +1,25 @@
 .gem-c-big-number {
   margin-bottom: govuk-spacing(3);
+  font-family: $govuk-font-family;
 }
 
 .gem-c-big-number__value {
-  @include govuk-font($size: 80, $weight: bold);
+  font-size: 80px;
+  font-weight: 700;
+  line-height: 1;
+
+  @if $govuk-typography-use-rem {
+    font-size: govuk-px-to-rem(80px);
+  }
 }
 
 .gem-c-big-number__label {
-  @include govuk-font($size: 16, $weight: bold, $line-height: 2);
+  font-size: 16px;
+  font-weight: 700;
```

Comment:
For similar reasons to above, I'd suggest using the [`govuk-typography-weight-bold`](https://frontend.design-system.service.gov.uk/sass-api-reference/#govuk-typography-weight-bold) mixin.